tox.ini

```diff
@@ -22,21 +23,24 @@ setenv =
 deps =
     pytest
     pytest-cov
-    flake8
-    mypy
     tox
 commands =
     pytest --basetemp={envtmpdir}
 
-[testenv:flake8]
-basepython = python3.7
-deps = flake8
-commands = flake8 click_repl tests
+[testenv:click6]
```

Comment:
why this is added?

Reply:
It seems like this module should support click v6, as there's an implementation of ClickExit Exception in the exceptions file

So, I've added click v6 tests as well